db/migrations/create_booking_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE bookings (bookingid SERIAL PRIMARY KEY, name VARCHAR(60), bookfrom DATE, bookto DATE, approved BOOLEAN, Userid INTEGER REFERENCES users(userid), Roomid INTEGER REFERENCES rooms(roomid));
```

Comment:
We can pull the room name by joining to the rooms table rather than storing two copies